pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1506,21 +1503,23 @@ def on_fs_type_changed(self, combo):
         self.on_value_changed()
 
     def on_encrypt_toggled(self, widget):
+        self._encryptCheckbox.set_inconsistent(False)
         self._request.device_encrypted = self._encryptCheckbox.get_active()
         self.on_luks_version_changed(self._luksCombo)
         self._update_luks_combo()
         self.on_value_changed()
 
     def _update_luks_combo(self):
-        hide_or_show = really_show if self._encryptCheckbox.get_active() else really_hide
-
-        for widget in [self._luksLabel, self._luksCombo]:
-            hide_or_show(widget)
+        visible = self._encryptCheckbox.get_active() or self._encryptCheckbox.get_inconsistent()
+        sensitive = self._encryptCheckbox.get_active()
```

Comment:
Move this line under `if visible:` part. It's not used elsewhere.

Reply:
Thanks to the suggestion, but this looks more readable to me.